promptshell/alias_manager.py

```diff
@@ -193,6 +194,25 @@ def expand_alias(self, input_command):
             base_command = self.aliases[alias_name]['command']
             return f"{base_command} {args}".strip()
         return input_command
+    
+    def clear_all_alias(self):
+        confirm = questionary.confirm("Are you sure you want to remove all aliases? (yes/no)").ask()
```

Comment:
```suggestion
        confirm = questionary.confirm("Are you sure you want to remove all aliases?").ask()
```
Remove the `(yes/no)` since questionary already prompts user to press y or n

Reply:
@Kirti-Rathi  I have made the changes.